app/Observers/DatasetVersionObserver.php

```diff
@@ -68,6 +66,9 @@ public function elasticDatasetVersion(DatasetVersion $datasetVersion)
         }
 
         if (!is_null($dataset) && $dataset->status === Dataset::STATUS_ACTIVE) {
+            ExtractPublicationsFromMetadata::dispatch($datasetVersion->id);
+            ExtractToolsFromMetadata::dispatch($datasetVersion->id);
```

Comment:
this now calls a reindex of tool - there is no dependancy - e.g. tool index has a dependency on publication that is modified by ExtractPublicationsFromMetadata? I think probably not...